README.md

```diff
@@ -18,7 +18,7 @@ Declare the plugin dependency in the _build.gradle_ file, as shown here:
 ```groovy
 dependencies {
     ...
-    compile("org.grails.plugins:sentry:7.8.1")
+    compile("org.grails.plugins:sentry:Sentry-1.2.0")
```

Comment:
I find it strange to add`Sentry-` to version number.
If want to keep version number, we could prepend `1` to sentry client, so next release would be `11.2.0`.
What do you think?